DESCRIPTION

```diff
@@ -44,3 +45,5 @@ Suggests:
     testthat,
     RcppArmadillo,
     RcppXPtrUtils
+RoxygenNote: 6.1.0
```

Comment:
Not sure why `RoxygenNote` was added since this package doesn't use roxygen.

Reply:
This might have gotten added automatically as I was running `devtools::check`. Should I submit a new PR removing that line?

Reply:
Yes, you can remove `RoxygenNote` and `Encoding` entry, since these are not needed (no need for new PR, just use a new commit or amend/rebase).

Reply:
removed, thanks